client/rest-high-level/src/main/java/org/opensearch/client/IndicesRequestConverters.java

```diff
@@ -749,8 +755,7 @@ static Request templatesExist(IndexTemplatesExistRequest indexTemplatesExistRequ
     }
 
     static Request templatesExist(ComposableIndexTemplateExistRequest indexTemplatesExistRequest) {
-        final String endpoint = new RequestConverters.EndpointBuilder()
-            .addPathPartAsIs("_index_template")
+        final String endpoint = new RequestConverters.EndpointBuilder().addPathPartAsIs("_index_template")
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Similar code fragments were detected in the same file at the following lines: 758:766, 734:742.  
Refactoring can help improve code maintainability. Consider reducing duplicate code by extracting it into a separate method. You can then replace duplicated code with calls to this new method.